### DISCUSSION TITLE
No surface temperature calculations using MRT system [Eddy 0.4.8.0]

### DISCUSSION THREAD

**Discussion by FlorenRoosen** [Eddy3D-Dev/Discussions#6] [Q&A]
Hello,

I am trying to run the MRT simulation system using the Eddy3d beta 0.4.8.0.

Everything seems fine, but it turns out that the simulation does not calculate the surface temperatures, when trying to inspect them, it does not provide any data. Furthermore, there is no significative difference in the MRT between different surfaces.

![image](https://github.com/user-attachments/assets/6b0e8d64-15fb-4688-b276-b87997dce779)

As I understood from [[https://www.sciencedirect.com/science/article/abs/pii/S0360132321010301]](https://www.sciencedirect.com/science/article/abs/pii/S0360132321010301) it seems like the simulation system is taking a shortcut towards assigning ambient temperature to surfaces / considering that it should not calculate surface temperatures (despite that I set computeSurfacetemperature to True ; and every radiation surfaces are on "simulated").

Where does the problem comes from ? 

The .gh file is attached for inspection : 
[EDDY-RADIANCE.zip](https://github.com/user-attachments/files/17879359/EDDY-RADIANCE.zip)

Thanks for your time,

Florent

**Comment by kastnerp** [comment]
Related: https://discourse.mcneel.com/t/mrt-component-crash/173809

**Comment by kastnerp** [comment]
Should be fixed in 0.5.0.815 (January 12, 2026). Unit tests pass.
